authors/apps/authentication/renderers.py

```diff
@@ -1,26 +1,12 @@
 import json
 
 from rest_framework.renderers import JSONRenderer
+from authors.apps.core.renderers import AuthorsJSONRenderer
 
 
-class UserJSONRenderer(JSONRenderer):
-    charset = 'utf-8'
+class UserJSONRenderer(AuthorsJSONRenderer):
+    object_label = 'user'
 
     def render(self, data, media_type=None, renderer_context=None):
-        # If the view throws an error (such as the user can't be authenticated
```

Comment:
I had some changes in this part of the application as well on the ft-password-reset-160609552 branch but changed the key-word being returned from 'users' to 'messages'. May I know what this change does?

Reply:
The user renderer had some shared code with the profiles renderer so I created a renderers file in the core app and put shared code there. This is now a sub class of the core renderer